test/integration/status.test.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright (c) 2017, Kinvey, Inc. All rights reserved.
+ *
+ * This software is licensed to you under the Kinvey terms of service located at
+ * http://www.kinvey.com/terms-of-use. By downloading, accessing and/or using this
+ * software, you hereby accept such terms of service  (and any agreement referenced
+ * therein) and agree that you have read, understand and agree to be bound by such
+ * terms of service and are of legal age to agree to such terms with Kinvey.
+ *
+ * This software contains valuable confidential and proprietary information of
+ * KINVEY, INC and is subject to applicable licensing agreements.
+ * Unauthorized reproduction, transmission or distribution of this file and its
+ * contents is a violation of applicable laws.
+ */
+
+const sinon = require('sinon');
+const chalk = require('chalk');
+
+const constants = require('./../../lib/constants');
+const logger = require('./../../lib/logger');
+
+const command = require('./../fixtures/command.js');
+const MockServer = require('./../mock-server');
+const helper = require('./../helper');
+
+describe('status', () => {
+  const mockServer = new MockServer(true);
+  const sandbox = sinon.sandbox.create();
+
+  const cmdStatusPath = './../../cmd/status';
+
+  afterEach((cb) => {
+    sandbox.restore();
+    MockServer.clearAll();
+    helper.setup.clearRequireCache();
+    helper.setup.clearUserProjectSetup(cb);
+  });
+
+  describe('when user and project are already set', () => {
+    beforeEach('setupUserAndProject', (cb) => {
+      helper.setup.configureUserAndProject(sandbox, mockServer, cb);
+    });
+
+    describe('setup is valid', () => {
+      it('should return status', (cb) => {
+        const serviceStatus = constants.ServiceStatus.ONLINE;
+        mockServer.serviceStatus(serviceStatus);
+
+        const spyLogger = sandbox.spy(logger, 'info');
+
+        // TODO: assert status when cb starts receiving it
+        require(cmdStatusPath)(command, (err, actualStatus) => {
+          expect(err).to.not.exist;
+          // expect(actualStatus).to.exist;
```

Comment:
If the callback starts receiving the status, these comments would be helpful when editing the tests IMO. Is there a policy against comments in production? Otherwise, they are mentioned in the style guide as an option https://github.com/Kinvey/style-guides/blob/master/backend/es6.md#comments--actionitems

Reply:
Sure, sounds good, but then `// expect(actualStatus)...` should be prefixed with `FIXME` or `TODO`